sentry/src/test/java/io/sentry/SentryTest.kt

```diff
@@ -76,6 +76,38 @@ class SentryTest {
         verify(scopes).close(eq(true))
     }
 
+    @Test
+    fun `global client is enabled after restart`() {
+        val scopes = mock<IScopes>()
+        whenever(scopes.close()).then { Sentry.getGlobalScope().client.close() }
+        whenever(scopes.close(anyOrNull())).then { Sentry.getGlobalScope().client.close() }
+
+        Sentry.init {
+            it.dsn = dsn
+        }
+        Sentry.setCurrentScopes(scopes)
+        Sentry.init {
+            it.dsn = dsn
+        }
+        verify(scopes).close(eq(true))
+        assertTrue(Sentry.getGlobalScope().client.isEnabled)
```

Comment:
Should we also check that `Sentry.getCurrentScopes()` has an enabled client?

Reply:
We basically already are. It'd be more of a duplicate of tests already present in ScopesTest / CombinedScopeViewTest. I think it's fine as is.